### DISCUSSION TITLE
Create a Alpha channel on TIF files

### DISCUSSION THREAD

**Discussion by joaodaffonsojr** [ImageMagick/ImageMagick#6624] [Help]
### ImageMagick version

ImageMagick-7.1.0-Q16-HDRI

### Operating system, version and so on

Windows 11

### Description

Hello !

I have two tiff files 01.tif and 02.tif
They are both grayscale.
I need to create a new channel name it "Alpha" on 01.tif and them open 02.tif copy and paste on new "Alpha" channel on 01.tif

any help is very welcome !

**Comment by snibgo** [comment]
I'm not sure what you want.
```
magick 01.tif 02.tif -compose CopyOpacity -composite out.tif
```
This reads your input files and writes to out.tif. The output will contain the colour channels from 01.tif, and an alpha channel. The alpha will come from the alpha of 02.tif if it has an alpha channel. Otherwise, it will come from the grayscale of 02.tif.

Does that help?

**Reply by joaodaffonsojr** [reply]
no it s not. 
In Photoshop is this that I need. 
two channels.
1 named "Gray" that should be 01.tif file
the other named "Alpha 1" and should be 02.tif file
![a1](https://github.com/ImageMagick/ImageMagick/assets/82617352/5ffe22c1-3571-4b3e-86d3-989c0062e5dc)
![a2](https://github.com/ImageMagick/ImageMagick/assets/82617352/4564867a-7521-4297-8035-34fc6af935f0)
![a3](https://github.com/ImageMagick/ImageMagick/assets/82617352/915bd2ca-8b9f-4073-9f5e-6e67137d0900)

**Comment by snibgo** [comment]
I don't use Photoshop, but I don't think IM has a method for naming channels.
